ci/ci.el6/integration_test/multibox/ind-steps/step-box/preconfigure.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+run_cmd "iptables -F"
```

Comment:
Actually I think this step can be removed altogether. I'm pretty sure there's no rules in iptables since there's no firewalld in centos 6. Not a big issue though.